### ISSUE TITLE
open folder

### ISSUE THREAD

**Issue by pebble2050** [d2phap/ImageGlass#1272]
Great work! It would be even greater if support open folder with sub folders. Since I have too many photos classified by folders.
Thanks.

**Comment by d2phap** [comment]
You can check this option:
![image](https://user-images.githubusercontent.com/3154213/150899403-6647347d-ec3e-4758-a6dd-3730c0bbdfed.png)

**Comment by pebble2050** [comment]
@d2phap 
Thanks for reply. Could you please add a menu for open fold. It would more convenient to open fold explicitly.  Sometimes It won't be a photo at root directory.

**Comment by d2phap** [comment]
Just changed this to feature request. While waiting for it to be implemented, you can drag and drop the folder to the viewer for now.

**Comment by BradKML** [comment]
Folder View Clarification: is it something like this? (Image from Qimgv)
![](https://raw.githubusercontent.com/easymodo/qimgv/master/qimgv/distrib/screenshots/qimgv0.9_2.jpg)

Also how would this tie in to https://github.com/d2phap/ImageGlass/issues/1067

**Comment by pebble2050** [comment]
@d2phap 
Yes,It seems OK.  FILESYSTEM(and the entire left side) tree should be togglable(show or not). May be add group display,When have sub directories in the right side. Thanks.

**Comment by d2phap** [comment]
ImageGlass is designed to work with File Explorer, having the folder tree creates many other following sub-features.

The "Open folder" I mean is this dialog:

![image](https://user-images.githubusercontent.com/3154213/158003151-844c3d3e-c21b-4c21-9c65-25e2b9bed903.png)

**Comment by BradKML** [comment]
@d2phap I guess large thumbnail rendering (or lack of ability to show multiple images in ImageGlass) is another issue entirely.

**Comment by HCentric** [comment]
Hi @d2phap,
I understand that as part of the project you do not allow the option that in addition to showing images selected in Windows Explorer will be able to select folders in ImageGlass in the future?
The togglable folder hierarchy that @BrandonKMLee  proposes is a pretty classic solution that gives a lot of possibilities for the future. 
All the best

**Comment by d2phap** [comment]
Hi @HCentric 

ImageGlass is designed for viewing and working with File Explorer only, that's why it lacks of photos editing and organizing features.

However, I will probably add support for simple browsing features after version 9 is complete.

**Comment by BradKML** [comment]
@d2phap a major issue is that file explorer may not be equipped to handle larger sized thumbnails or previews on Windows, and that "get all images from a folder" is not easy on it.
